rusk-schema/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rusk-schema"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
This should be 0.1.1
According to gRPC protocol, adding a field to a response is considered a non-breaking change

_Fields added to a response message are deserialized into the message's unknown fields collection on the client._

Reply:
I'm aware of this, however this is also a crate, and the addition of a field to a struct breaks instantiation.